cms/templatetags/image_version_url.py

```diff
@@ -0,0 +1,34 @@
+"""CMS templatetags"""
+
+from urllib.parse import quote_plus, urljoin
+from django import template
+from django.conf import settings
+from wagtail.images.templatetags.wagtailimages_tags import image_url
+
+register = template.Library()
+
+
+@register.simple_tag()
+def image_version_url(
+    image, filter_spec, full_url=False, viewname="wagtailimages_serve"
+):
+    """
+    Generates an image URL using Wagtail's library and appends a version to the path to enable effective caching
```

Comment:
I think we created this for caching wagtail images + image optimizations. And for that, we also used wagtail caching backend settings that can be seen [here in xPRO](https://github.com/mitodl/mitxpro/blob/master/mitxpro/settings.py#L912-L927). We have seen some issues with image caching though. So, I'm not sure if we want to introduce caching here or just use `image_url` from `wagtailimages_tags` in HTML.

Reply:
I don't want to introduce image caching as a side-effect of this PR. And there's nothing about certificates by themselves that recommends caching. 

Can we leave it out without to much trouble?